### ISSUE TITLE
Can't connect to JanusGraph on Compose

### ISSUE THREAD

**Issue by bogumil** [jbmusso/gremlin-javascript#90]
I'm getting `Error: Failed to authenticate (Error 401)` when trying to connect to a server using username, password and ssl.

It looks that Basic Authentication is required - based on https://help.compose.com/docs/janusgraph-connecting-to-janusgraph#section-the-web-socket-connection-strings

**Comment by jbmusso** [comment]
Indeed, this is not supported yet. If you have time to provide a pull request for this, I'll be happy to review it. This is something that we should support as well.

**Comment by pluradj** [comment]
I've confirmed this now works with version `2.7.0`
